tarantool/connection.py

```diff
@@ -319,6 +390,8 @@ def connect(self):
         '''
         try:
             self.connect_basic()
+            if self.transport == SSL_TRANSPORT:
+                self.wrap_socket_ssl()
```

Comment:
The reason is same as why was split a socket connection and IPROTO handshake. More clear behavior in that case:

https://github.com/tarantool/tarantool-python/blob/46907c7f2daa603e6e8e62a20c9ca0edfc6f8331/tarantool/connection.py#L435-L450

Reply:
If `connect_basic()` will raise before TLS wrapping, we'll repeat only plain TCP connection. Or you're about possible error from TLS handshake: we don't need extra code to avoid repeating those errors?

Okay, I don't mind.